Doc/cmus.txt

```diff
@@ -150,47 +151,46 @@ prepend tracks to the play queue (4)
 
 @h2 Moving Tracks
 
-In views 2-4 you can move tracks within the list. Note that moving is
-disabled if the view is auto-sorted (see *lib_sort* and *pl_sort* options).
+In views 2-4, tracks can be moved within the list. Note that moving is disabled
+if the view is auto-sorted (see *lib_sort* and *pl_sort* options).
 
-Pressing *p* moves marked tracks to the position immediately after the
-selected track.  *P* moves them to the position immediately before the
-selected track.  If there are no marked tracks then the selected track is
-moved down (*p*) or up (*P*).
+Pressing *p* moves marked tracks to the position immediately after the selected
+track. *P* moves them to the position immediately before the selected track. If
+there are no marked tracks, the selected track is moved down (*p*) or up (*P*).
 
-NOTE: Changing active filters reloads view 2 so it isn't a good idea to
-manually order tracks in the view.
+Note that changing active filters in view 2 reloads it, losing any changes made
+to the track order.
 
 @h2 Removing Tracks
 
-Press *D* or *delete* to remove marked or selected tracks in the current view
-(1-4).  The tracks will be removed immediately from the view without asking
-for confirmation.  In the browser and filters views the same keys are used to
-remove a file or filter (will ask for confirmation).
+Press *D* or *delete* to remove the marked or selected tracks in the current
+view (1-4). The tracks will be removed immediately from the view without asking
+for confirmation. In the browser and filters views, the same keys are used to
+remove a file or filter after asking for confirmation.
 
 
 @h1 STATUS LINE
 
-Right hand side of the status line (second row from the bottom, black text on
-a grey background) consists of the following fields:
+The right hand side of the status line (second row from the bottom, black text
+on a grey background) consists of the following fields:
 
 @pre
 aaa_mode & play_sorted & play_library | continue follow repeat shuffle
 @endpre
 
-NOTE: *aaa_mode* and *play_sorted* will be displayed only if *play_library* is
+NOTE: *aaa_mode* and *play_sorted* will be only displayed if *play_library* is
 *true* because these are meaningless when playing the playlists (view 3).
 
-Pressing *m*, *o*, *M*, *C*, *r* and *s* keys should make it easier to
-understand what all those fields mean.
+Pressing *m*, *o*, *M*, *C*, *r* and *s* should make it easier to understand
+what these fields mean.
 
 See CONFIGURATION OPTIONS section for more information about these options.
 
 
 @h1 KEYBINDINGS
 
-Here's list of default keybindings.  See *unbind* and *bind* commands in the
-COMMANDS section.
+Here's list of default keybindings. To change them, see the *unbind* and *bind*
+commands in the COMMANDS section.
 
 
 @h2 Common Context
```

Comment:
By the way, are these bindings up to date?

Reply:
I'll check that. I'll probably also add a note about them being visible in the last view.

Reply:
I'll need to update them. I'll merge the ones from `rc` into the documentation, then I'll update `rc` to match the order of the documentation.

Reply:
Also, I'll need to add some missing commands to the documentation:

<details>

```
$ grep -E '[{] "[a-z_-]+",\s+cmd_' command_mode.c | cut -d\" -f2 | while read cmd; do printf "%-40s %s\n" "${cmd}" "$(grep -Em1 "^${cmd}\\s" Doc/cmus.txt || echo '!!!')"; done
add                                      add [-l] [-p] [-q] [-Q] <file|dir|url|playlist>
bind                                     bind [-f] <context> <key> <command>
browser-up                               browser-up (*backspace*)
cd                                       cd [directory]
clear                                    clear [-l] [-p] [-q]
colorscheme                              colorscheme <name>
echo                                     echo <arg>...
factivate                                factivate <user-defined-filter>...
filter                                   filter <filter-expression>
fset                                     fset <name>=<filter-expression>
help                                     !!!
invert                                   !!!
live-filter                              live-filter <simple-filter-expression|short-filter-expression>
load                                     load [-l] [-p] <playlist>
lqueue                                   lqueue [NUM]
mark                                     mark <filter-expression>
mute                                     !!!
player-next                              player-next (*b*)
player-pause                             player-pause (*c*)
player-pause-playback                    !!!
player-play                              player-play [filename] (*x*)
player-prev                              player-prev (*z*)
player-stop                              player-stop (*v*)
prev-view                                !!!
left-view                                !!!
right-view                               !!!
pl-create                                pl-create <name>
pl-export                                pl-export <filename>
pl-import                                pl-import [filename]
pl-rename                                pl-rename <name>
push                                     push [text]
pwd                                      !!!
raise-vte                                !!!
rand                                     !!!
quit                                     quit [-i] (*q*, *:wq*)
refresh                                  refresh (*^L*)
run                                      run <command>
save                                     save [-e]  [-l] [-L] [-p] [-q]  [file]  (*:w*)
search-b-start                           !!!
search-next                              search-next (*n*)
search-prev                              search-prev (*N*)
search-start                             !!!
seek                                     seek [+-](<num>[mh] | [HH:]MM:SS)
set                                      set <option>=<value>
shell                                    shell <command>
showbind                                 showbind <context> <key>
shuffle                                  shuffle (false)
source                                   source <filename>
toggle                                   toggle <option>
tqueue                                   tqueue [NUM]
unbind                                   unbind [-f] <context> <key>
unmark                                   !!!
update-cache                             update-cache [-f]
version                                  !!!
view                                     view <name or 1-7>
vol                                      vol [+-]NUM[%] [[+-]NUM[%]]
w                                        !!!
win-activate                             win-activate (*enter*)
win-add-l                                win-add-l [-n] (*a*)
win-add-p                                win-add-p [-n] (*y*)
win-add-q                                win-add-q [-n] (*e*)
win-bottom                               win-bottom (*G*, *end*)
win-down                                 win-down [NUM] (*j*, *down*)
win-half-page-down                       win-half-page-down (*^D*)
win-half-page-up                         win-half-page-up (*^U*)
win-mv-after                             win-mv-after (*p*)
win-mv-before                            win-mv-before (*P*)
win-next                                 win-next (*tab*)
win-page-bottom                          !!!
win-page-down                            win-page-down (*^F*, *page_down*)
win-page-middle                          !!!
win-page-top                             !!!
win-page-up                              win-page-up (*^B*, *page_up*)
win-remove                               win-remove (*D*, *delete*)
win-scroll-down                          win-scroll-down (*^E*)
win-scroll-up                            win-scroll-up (*^Y*)
win-sel-cur                              win-sel-cur (*i*)
win-toggle                               win-toggle (*space*)
win-top                                  win-top (*g*, *home*)
win-up                                   win-up [NUM] (*k*, *up*)
win-update                               win-update (*u*)
win-update-cache                         win-update-cache [-f]
wq                                       !!!
```

</details>

Reply:
There's a conflict between rc and the docs: doc=`i                               win-sel-cur` rc=`i toggle show_hidden`.

And, stuff in rc but not the docs:

```
u win-update
backspace browser-up
space win-activate
^U win-half-page-up
^D win-half-page-down
i win-sel-cur
k win-up
U win-update-cache
```

Reply:
I'm going to put this in another PR since this one is getting somewhat large.

Reply:
I've opened #1037 with these changes.